src/fsharp/LexFilter.fs

```diff
@@ -898,7 +935,7 @@ type LexFilterImpl (lightSyntaxStatus: LightSyntaxStatus, compilingFsLib, lexer,
                             // and insert a HIGH_PRECEDENCE_PAREN_APP
                             if not hasAfterOp && (match nextTokenIsAdjacentLParenOrLBrack lookaheadTokenTup with Some LPAREN -> true | _ -> false) then
                                 let dotTokenTup = peekNextTokenTup()
-                                stack := (dotTokenTup.UseLocation(HIGH_PRECEDENCE_PAREN_APP), false) :: !stack
+                                stack := (pool.UseLocation(dotTokenTup, HIGH_PRECEDENCE_PAREN_APP), false) :: !stack
```

Comment:
How would you feel about making `stack` a `let mutable` instead of a ref cell and modifying where it's set? Not related to this change, but generally I think it helps readability in the long run if we move away from ref cells

Reply:
It's fine that we can change that. 

In general, existing stylistic design I don't want to have the expectation of fixing them up in PRs. If it happens in the PR, then it's fine, but nit-picking what's already written because we made changes in that area shouldn't be the concern of the PR; if that's going to be the case then we could spend ages on a single PR going over existing styles.

Reply:
Covered in a different PR :)